addons/report/views/templates.xml

```diff
@@ -79,6 +79,16 @@
                 <t t-call-assets="web_editor.summernote" t-css="false"/>
                 <t t-call-assets="web_editor.assets_editor" t-css="false"/>
                 <t t-call-assets="report.assets_editor" t-css="false"/>
+                <script>
```

Comment:
It seems strange, why do you need to load this in the basic website.layout template ?